File: src/Ebay/Server.Application.New/Models/TubeWorkingPointInfo.fs
namespace Server.Application.New.Models

/// <summary>
/// Параметры рабочей точки лампы.
/// </summary>
type TubeWorkingPointInfo =
    {
        /// <summary>
        /// Анодное напряжение, В.
        /// </summary>
        AnodeVoltage: double

        /// <summary>
        /// Напряжение на сетке, В.
        /// </summary>
        GridVoltage: double

        /// <summary>
        /// Полуширина диапазона анодного напряжения, В.
        /// </summary>
        AnodeVoltageHalfWidth: double

        /// <summary>
        /// Полуширина диапазона напряжения сетки, В.
        /// </summary>
        GridVoltageHalfWidth: double

        /// <summary>
        /// Номинальный ток, мА.
        /// </summary>
        NominalCurrent: double
    }

File: src/Ebay/Server.Application.Abstractions.Driven/Models/TubeWorkingPointInfo.cs
namespace Server.Application.Abstractions.Driven.Models;

/// <summary>
/// Параметры рабочей точки лампы.
/// </summary>
/// <param name="AnodeVoltage">Анодное напряжение, В.</param>
/// <param name="GridVoltage">Напряжение на сетке, В.</param>
/// <param name="AnodeVoltageHalfWidth">Полуширина диапазона анодного напряжения, В.</param>
/// <param name="GridVoltageHalfWidth">Полуширина диапазона напряжения сетки, В.</param>
/// <param name="NominalCurrent">Номинальный ток, мА.</param>
public record TubeWorkingPointInfo(
    double AnodeVoltage,
    double GridVoltage,
    double AnodeVoltageHalfWidth,
    double GridVoltageHalfWidth,
    double NominalCurrent
);